ps_checkout.php

```diff
@@ -828,6 +828,15 @@ public function hookActionAdminControllerSetMedia()
         /** @var \PrestaShop\Module\PrestashopCheckout\Version\Version $version */
         $version = $this->getService('ps_checkout.module.version');
 
+        if ('AdminModules' === Tools::getValue('controller')) {
```

Comment:
```suggestion
        if ('AdminModules' === Tools::getValue('controller') && 'ps_checkout' === Tools::getValue('configure')) {
```
Avoid to load the file on every other module configuration page